app.rb

```diff
@@ -4,14 +4,10 @@
 require('./rental')
 require('./list')
 require('./create')
-
 class App
   def initialize
-    @people = []
-    @books = []
-    @rentals = []
-    @list = List.new(@books, @people, @rentals)
-    @create = Create.new(@books, @people, @rentals)
+    @list = List.new
+    @create = Create.new
```

Comment:
- [ ] **Required:** I have noticed that the `main.rb` file hasn't changed in this branch and so I cannot comment on it.

The app crashes when I attempt to create a `Person` (Student and Teacher) or `Book`. Kindly trace the errors from the gif video attached below.

![crashing_app](https://user-images.githubusercontent.com/27709832/183103563-b671fd2d-80f9-4627-b053-c8dd520911fb.gif)